dspace-xmlui/src/main/webapp/i18n/messages.xml

```diff
@@ -1024,6 +1029,7 @@
         <message key="xmlui.administrative.Navigation.context_export_collection">Export Collection</message>
         <message key="xmlui.administrative.Navigation.context_export_community">Export Community</message>
         <message key="xmlui.administrative.Navigation.context_export_metadata">Export Metadata</message>
+    <message key="xmlui.administrative.Navigation.context_liveimport">Elsevier Import</message>
```

Comment:
Hi Tim, Lieven,

Lieven, do the plugins also work for to integrate with the Scopus search API? If not, does it make sense to go with Tim’s suggestion to change “Elsevier” to “ScienceDirect”? I would think so.

Thanks for your advice.

Best regards,

Letitia Mukherjee
Market Development Manager Sharing Platforms
Mob: +31 6 13 98 98 71

From: Tim Donohue [mailto:notifications@github.com]
Sent: 26 January 2016 15:51
To: DSpace/DSpace
Cc: Mukherjee, Letitia (ELS-AMS)
Subject: Re: [DSpace] DS-2877 Import of ScienceDirect metadata including embargo and linking to or embedding of the final version (#1162)


In dspace-xmlui/src/main/webapp/i18n/messages.xml<https://github.com/DSpace/DSpace/pull/1162#discussion_r50843497>:

> @@ -1024,6 +1029,7 @@

>          <message key="xmlui.administrative.Navigation.context_export_collection">Export Collection</message>

>          <message key="xmlui.administrative.Navigation.context_export_community">Export Community</message>

>          <message key="xmlui.administrative.Navigation.context_export_metadata">Export Metadata</message>

> +    <message key="xmlui.administrative.Navigation.context_liveimport">Elsevier Import</message>

Quick question: I noticed the UI text mentions "Elsevier" in several places (this is just one of the instances). Would it not be more appropriate to call this "ScienceDirect Import"? Will DSpace end users know that "Elsevier Import" is actually pulling data from ScienceDirect?

At least to me, it seems more logical to rename all these to reference ScienceDirect (the product we are pulling data from), instead of referencing Elsevier (the company). There are more instances of "Elsevier" in the messages.xml, and it seems like in most or maybe all cases, it might make more sense to replace them with "ScienceDirect".

—
Reply to this email directly or view it on GitHub<https://github.com/DSpace/DSpace/pull/1162/files#r50843497>.

________________________________

Elsevier B.V. Registered Office: Radarweg 29, 1043 NX Amsterdam, The Netherlands, Registration No. 33156677, Registered in The Netherlands.